libraries/math-utils/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "spl-math-utils"
```

Comment:
Can we have this crate be `spl-math` and change the one with the program to `spl-math-example`?

Reply:
Done. Set the spl-math version back to 0.3 and set spl-math-example version to 0.1